src/map/script.cpp

```diff
@@ -26524,6 +26535,7 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(laphine_synthesis, ""),
 	BUILDIN_DEF(laphine_upgrade, ""),
 	BUILDIN_DEF(randomoptgroup,"i"),
+	BUILDIN_DEF(open_quest_ui, "?"),
```

Comment:
Here and in script documentation you made quest id optional, but in reality it is mandatory.
Maybe make it optional for real - if the client supports sending 0 as quest id.

Reply:
The client just open the UI when the quest id is 0

Reply:
Nice! :)